metron-platform/metron-common/src/main/java/org/apache/metron/common/stellar/StellarCompiler.java

```diff
@@ -40,14 +40,22 @@
 import static java.lang.String.format;
 
 public class StellarCompiler extends StellarBaseListener {
-  private static Token<?> EXPRESSION_REFERENCE = new Token<>(null, Object.class);
-  private static Token<?> LAMBDA_VARIABLES = new Token<>(null, Object.class);
+  private static Token<?> EXPRESSION_REFERENCE = new Token<>(null, Object.class, null);
+  private static Token<?> LAMBDA_VARIABLES = new Token<>(null, Object.class, null);
 
   private Expression expression;
   private final ArithmeticEvaluator arithmeticEvaluator;
   private final NumberLiteralEvaluator numberLiteralEvaluator;
   private final ComparisonExpressionWithOperatorEvaluator comparisonExpressionWithOperatorEvaluator;
 
+  public static class ShortCircuitFrame {}
+  public static class ShortCircuitOp {}
```

Comment:
We are actually instantiating them in this class.  These are used to indicate an instance of a short circuit frame on the stack.  Unlike some of the other marker classes, we actually want separate instances of these because you could have nested frames.